app/src/androidTest/java/com/cesar/knot_thing_lib_android/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.cesar.knot_thing_lib_android
```

Comment:
please remove this class if you're not testing anything.

Reply:
Done.